pages/auth-pages/forgot-password.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet" />
+    <link rel="stylesheet" href="https://orion-ui.netlify.app/components/orion-ui.css" />
+    <link rel="stylesheet" href="/style.css" />
+    <link rel="shortcut icon" href="/assets/favicon.ico" type="image/x-icon" />
+    <title>Signin</title>
+  </head>
+  <body>
+    <!-- nav-bar -->
+    <nav class="nav">
+      <header class="nav-header">
+        <a class="nav-heading text-lg" href="/index.html"><h2>Orion</h2></a>
+      </header>
+
+      <ul class="inline-style-list no-style-list nav-list">
+        <li>
+          <a class="nav-link text-primary-color" href="index.html"> Home </a>
+        </li>
+        <li>
+          <a class="btn btn-outline-primary" href="#">
+            <i class="fas fa-sign-out"></i>
+          </a>
+        </li>
+
+        <li>
+          <a class="btn btn-primary">Login</a>
+        </li>
+      </ul>
+    </nav>
+    <!-- --------------------------- -->
+
+    <!-- password-assistance-form -->
```

Comment:
can we remove the unwanted comments?

Reply:
Actually, I made those comments intentionally for separation. But thanks for the review.